src/Middleware/TraceMiddleware.php

```diff
@@ -74,7 +67,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         if (! empty($this->config['ignore_path']) && preg_match($this->config['ignore_path'], $request->getUri()->getPath())) {
             return $handler->handle($request);
         }
-
+        $tracer = TracerContext::getTracer();
```

Comment:
```suggestion
        $this->tracer = TracerContext::getTracer();
```
Esse ajuste aqui é necessário para corrigir esse erro aqui: 
```
Typed property Hyperf\Tracer\Middleware\TraceMiddleware::$tracer must not be accessed before initialization
```